### DISCUSSION TITLE
Hello, update to kernel 6.8.8-1 broke my (Broadcom) LAN/NIC.

### DISCUSSION THREAD

**Discussion by limsandy** [tteck/Proxmox#3240] [Help Needed]
Hello, it was only a few days ago that I ran this post-pve-install.sh and got updated to kernel 6.8.4-3 and life was perfect.

I had to format my system and today I ran the same post-pve-install.sh and noticed that my LAN/NIC broke and my kernel is now updated 6.8.8-1.

Without much troubleshooting, how do I run post-pve-install.sh but update up to 6.8.4-3 only?

**Comment by tteck** [comment]
Try pinning the old kernel to see if that resolves the issue.
https://tteck.github.io/Proxmox/#proxmox-ve-kernel-pin

**Reply by limsandy** [reply]
I have tried that, my network access is broken after the update.

Pinning/loading the older kernel (6.8.4-2) doesn't make it work.

But I can assure you my network card is fine before the update.

Re-install proxmox and don't update? I kinda need my system up and running ASAP. :(

**Reply by tteck** [reply]
Did your network card get renamed?

**Reply by limsandy** [reply]
Sorry, I'm new and don't understand how it could get renamed. Maybe automatically, but I didn't do it manually.

So what I did was after updating with your script, I shut down my computer, switch the OS disk (with old Proxmox inside it) with another SSD I had to do some operations. And after I was done, I switched back to the original SSD with new Proxmox.

**Reply by limsandy** [reply]
Do you have a quick fix please?

If not, I'm thinking of starting fresh again and maybe don't run your script and don't update?

**Reply by tteck** [reply]
Why not check if the update changed the names?

https://forum.proxmox.com/threads/latest-proxmox-update-8-2-2-renamed-network-interfaces.145925/

**Reply by limsandy** [reply]
No, there were some codes, but I quote my network interface as enp9s0, is that good or bad?

**Reply by limsandy** [reply]
I don't know what it was before the update.

**Reply by limsandy** [reply]
Is there any way to limit update to 6.8.4-3 instead of 6.8.8-1 ?

How do I fix my network interface please?

Thank you in advance for helping me.....

**Reply by limsandy** [reply]
Hello @tteck , do you plan to disable update to 6.8.8-1 as it seems like it's causing the damage (presumably not only to me, but everyone who updates) and limit the update to 6.8.4-3 ?

**Reply by tteck** [reply]
No, I have had no issues.

**Reply by tteck** [reply]
You are listing the kernel version. You can pin a kernel version.

**Reply by limsandy** [reply]
I believe it was the update that broke my network but I'm not 100% sure. I am trying to update again because my amdgpu driver install won't work otherwise.

**Reply by limsandy** [reply]
I updated again and kept my network. Yeay!
